test/e2e/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"flag"
+	"github.com/golang/glog"
+	osconfigv1 "github.com/openshift/api/config/v1"
```

Comment:
this is to test the operator status which I can't see  here, hence don't need to vendor. So either bring the test for it here or remove revendoring

Reply:
removed